From 5da9863bbd5d807cdedf82b5fbd4a6c003c6052c Mon Sep 17 00:00:00 2001
From: Sandor Yu <R01008@freescale.com>
Date: Fri, 3 Apr 2015 15:37:08 +0800
Subject: [PATCH] MLK-10575: clk: Remove IMX7D_PLL_VIDEO_POST_DIV from
 clks_init_on

CLK_SET_PARENT_ON flag have added when clk register,
IMX7D_PLL_VIDEO_POST_DIV will been turn on when
IMX7D_LCDIF_PIXEL_ROOT_SRC turn on,
So remove IMX7D_PLL_VIDEO_POST_DIV from clks_init_on.

Signed-off-by: Sandor Yu <R01008@freescale.com>
---
 arch/arm/mach-imx/clk-imx7d.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-imx/clk-imx7d.c b/arch/arm/mach-imx/clk-imx7d.c
index 42e0e9e..be4edbf 100644
--- a/arch/arm/mach-imx/clk-imx7d.c
+++ b/arch/arm/mach-imx/clk-imx7d.c
@@ -369,7 +369,7 @@ static int const clks_init_on[] __initconst = {
 	IMX7D_PLL_SYS_PFD4_CLK, IMX7D_PLL_SYS_PFD5_CLK, IMX7D_PLL_SYS_PFD6_CLK,
 	IMX7D_PLL_SYS_PFD7_CLK, IMX7D_DRAM_PHYM_ROOT_CLK, IMX7D_DRAM_ROOT_CLK,
 	IMX7D_DRAM_PHYM_ALT_ROOT_CLK, IMX7D_DRAM_ALT_ROOT_CLK,
-	IMX7D_AHB_CHANNEL_ROOT_CLK, IMX7D_NAND_USDHC_BUS_ROOT_CLK, IMX7D_PLL_VIDEO_POST_DIV,
+	IMX7D_AHB_CHANNEL_ROOT_CLK, IMX7D_NAND_USDHC_BUS_ROOT_CLK,
 	};
 
 static struct clk_onecell_data clk_data;
-- 
1.8.0

